lib/fizzy/parser.hpp

```diff
@@ -34,6 +34,18 @@ inline parser_result<uint8_t> parse_byte(const uint8_t* pos, const uint8_t* end)
     return {*pos, pos + 1};
 }
 
+template <typename T>
+inline parser_result<T> parse_value(const uint8_t* pos, const uint8_t* end)
```

Comment:
Since this needs to be rebased, you may want to update the commit message to read `parse_value` insead of `read_const`, but it is not important.

Reply:
It's "move and rename `read_const`" I would say